packages/headless/src/features/analytics/search-analytics-actions-loader.ts

```diff
@@ -461,7 +461,7 @@ export interface SearchAnalyticsActionCreators {
    * @returns A dispatchable action.
    */
   logExpandSmartSnippetSuggestion(
-    payload: LogSmartSnippetSuggestionActionCreatorPayload
+    payload: QuestionAnsweringDocumentIdentifierActionCreatorPayload
```

Comment:
Would it be possible to shorten the interface name by adjusting `Identifier` -> `Id`?